q2_feature_table/_summarize/tabulate_seqs_assets/index.html

```diff
@@ -8,6 +8,10 @@
         sequence and then click the <i>View report</i>
         button on the resulting page.
       <p>
+      <p>
+        To download a raw FASTA file of your sequences, click
+        <a href="sequences.fasta" download="sequences.fasta">here</a>.
```

Comment:
Same, actually. Do you know if there is a fallback for non-Chrome/Firefox/HTML5 browsers, or is this safe enough for us?